runtime/compiler/control/MongoLogger.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef MONGOLOGGER_HPP
+#define MONGOLOGGER_HPP
+
+#include "BasePersistentLogger.hpp"
+#include "LoadDBLibs.hpp"
+
+class MongoLogger : public BasePersistentLogger
+   {
+private:
+   //TODO: Allocate this with OpenJ9 Allocators.
```

Comment:
If it's statically allocated you don't need allocators.The space is big though, so there is a danger of stack overflow. Maybe reducing that to 256 is sufficient.